glean-core/src/metrics/timing_distribution.rs

```diff
@@ -247,13 +265,15 @@ impl TimingDistributionMetric {
                 if sample < 0 {
                     num_negative_samples += 1;
                 } else {
-                    let sample = sample as u64;
-                    let mut sample = self.time_unit.as_nanos(sample);
+                    let mut sample = sample as u64;
```

Comment:
Maybe we should call it out that `sample` is already in `time_unit` and thus we can compare it directly to the `MAX_SAMPLE_TIME`

Reply:
Good idea.  Added comment.